AppServer/google/appengine/api/urlfetch_stub.py

```diff
@@ -376,6 +379,9 @@ def _RetrieveURL(url, payload, method, headers, request, response,
                 ca_certs=CERT_PATH)
           else:
             connection_class = httplib.HTTPSConnection
+            # Disable native certificate verification.
+            if hasattr(ssl, '_create_unverified_context'):
+              use_unverified_context = True
```

Comment:
Python 2.7.9 and above validates certificates by default. This disables that behavior in cases when validation is not requested. We do something similar with our soap clients, but it's a little bit cleaner here since it doesn't affect other connections.